Training/Repository/CowRepository.cs

```diff
@@ -13,15 +13,16 @@ public Task Delete(Cow entity) {
             throw new NotImplementedException();
         }
 
+        public async Task<Cow> DeleteWithoutDeleteing(Guid id)
+            => await _context.DeleteWithoutDeleting<Cow>(id);
+
         public async Task<IEnumerable<Cow>> GetAll()
-            => await _context.AllSqlEntitites<Cow>().ToListAsync();
+            => await _context.AllSqlEntitites<Cow>().Include(c => c.Farmer).ToListAsync();
```

Comment:
why is this needed?

Reply:
because it shows null in the get if its not added

Reply:
and what is the problem? you are querying the db unnecessarily